sections/futures/MarketInfoBox/MarketInfoBox.tsx

```diff
@@ -19,20 +25,82 @@ const MarketInfoBox: React.FC<MarketInfoBoxProps> = ({
 	buyingPower,
 	marginUsage,
 	isMarketClosed,
+	potentialTrade,
+	maxLeverageValue,
```

Comment:
One issue I'm seeing is that this is not the actual "max leverage" for the market. This is the max leverage available to the user.

When I say "max leverage" I mean 10x, however when you have a position open this value is <10. This is causing issues, and you'll need to figure out how to get the actual static maximum in this component.